e2e/tests/localAuthorityMaintainsCaseAfterGatekeeping_test.js

```diff
@@ -36,6 +36,6 @@ Before(async (I, caseViewPage, submitApplicationEventPage, enterFamilyManCaseNum
   await I.navigateToCaseDetails(caseId);
 });
 
-Scenario('local authority uploads documents', uploadDocs.uploadDocuments());
+Scenario('local authority uploads documents', uploadDocs.assertMandatoryDocuments(), uploadDocs.uploadMandatoryDocuments());
```

Comment:
Looks like it is in the wrong order, would it be possible to maybe change it to read something like
```js
Scenario('local authority uploads documents', async (I, caseViewPage, uploadDocumentsEventPage) => {
  // navigate to event...
  uploadDocs.uploadMandatoryDocuments()
  // complete event...
  // navigate to tab...
  uploadDocs.assertMandatoryDocuments()
});
```

Reply:
Works right to left. Yeah was not sure wether we wanted to stick to the one line methods. I'll expand them out.